Settings.py

```diff
@@ -0,0 +1,2 @@
+db_path = r"C:\Users\tali pruss\Desktop\scores.db"
+db_table_name = r"scores"
```

Comment:
in this case:db_path = r"C:\Users\tali pruss\Desktop\scores.db"
because we have the space in the words "tali pruss" I must use the r or it's not working